glide.yaml

```diff
@@ -20,3 +20,5 @@ import:
 - package: github.com/arschles/kubeapp
   subpackages:
   - api
+- package: github.com/docker/docker
+  version: 2b27fe17a1b3fb8472fde96d768fa70996adf201
```

Comment:
Try a fresh glide install from what was there before... you'll find it fails because the sha its pinned to does not exist in docker/docker. I resolved this by pinning docker/docker to its previous value. Note I did put this in its own commit.